### ISSUE TITLE
Consolidate styles of focus state

### ISSUE THREAD

**Issue by stereobooster** [contiamo/operational-ui#1006]
We implemented 2 new components (Accordion, Tabs) with slightly updated style. And there is new PR https://github.com/contiamo/operational-ui/pull/1005. Those PR add 2 new styles for focus, as well we have style of focus from previous code.

We need to use one consistent style for focus state.

<img width="382" alt="Screenshot 2019-06-05 at 17 43 20" src="https://user-images.githubusercontent.com/179534/58970586-03dd1c80-87ba-11e9-83be-c1d4fa06dd85.png">
<img width="124" alt="Screenshot 2019-06-05 at 17 43 41" src="https://user-images.githubusercontent.com/179534/58970589-0475b300-87ba-11e9-8134-921edb8121c7.png">
<img width="203" alt="Screenshot 2019-06-05 at 17 43 54" src="https://user-images.githubusercontent.com/179534/58970590-0475b300-87ba-11e9-8c92-8b84cd573f3d.png">
<img width="342" alt="Screenshot 2019-06-05 at 17 44 59" src="https://user-images.githubusercontent.com/179534/58970591-0475b300-87ba-11e9-80d8-4280f28aacea.png">

After #1005
<img width="225" alt="Screenshot 2019-06-05 at 17 46 35" src="https://user-images.githubusercontent.com/179534/58970593-050e4980-87ba-11e9-8ab9-507930dd23d9.png">
<img width="202" alt="Screenshot 2019-06-05 at 17 46 40" src="https://user-images.githubusercontent.com/179534/58970594-050e4980-87ba-11e9-8630-dd701fb14b05.png">
<img width="176" alt="Screenshot 2019-06-05 at 17 46 49" src="https://user-images.githubusercontent.com/179534/58970595-050e4980-87ba-11e9-8e3e-c15da14e5682.png">

**Comment by stereobooster** [comment]
As well we need to show focus only for keyboard users, but not for mouse users (like described here https://css-tricks.com/keyboard-only-focus-styles/)

**Comment by TejasQ** [comment]
This is a great issue.

**Comment by michaeldera** [comment]
Is this still available to work on?

**Comment by TejasQ** [comment]
There isn't really an idea of what to implement here yet – it's more a design question of which style we settle on. @kemal-contiamo or @micha-f any ideas? 

When we decide which one wins, @michaeldera, would you like to implement it?

**Comment by michaeldera** [comment]
Yes please. Would love to tackle it.

**Comment by TejasQ** [comment]
You got it! Let's wait for their feedback.

**Comment by kemal-contiamo** [comment]
> There isn't really an idea of what to implement here yet – it's more a design question of which style we settle on. @kemal-contiamo or @micha-f any ideas?
> 
> When we decide which one wins, @michaeldera, would you like to implement it?

@TejasQ lets streamline our focus state to the ones that Slava already showed above, which is these:
<img width="239" alt="Screenshot 2019-06-24 at 11 24 19" src="https://user-images.githubusercontent.com/49938430/60007533-c100bd00-9672-11e9-9298-6d91b6a4bafc.png">
<img width="170" alt="Screenshot 2019-06-24 at 11 24 23" src="https://user-images.githubusercontent.com/49938430/60007534-c100bd00-9672-11e9-95fd-32d68148679b.png">
In other words: transform the colour of the border line on the focused element to #1499CE (hex code)

**Comment by TejasQ** [comment]
@michaeldera you got that?

**Comment by mpotomin** [comment]
Please ensure, that the updated focus styles make Action menu look focused for the Table example with actions (more details in this PR https://github.com/contiamo/operational-ui/pull/1068)

**Comment by michaeldera** [comment]
> @michaeldera you got that?

Got it!

**Comment by michaeldera** [comment]
> Please ensure, that the updated focus styles make Action menu look focused for the Table example with actions (more details in this PR [#1068 (comment)](https://github.com/contiamo/operational-ui/pull/1068#issuecomment-504908515))

I may not be getting this correctly.. So I am working on this right now and the actions do look focused (I am assuming that looking focused is the blue color here).. is that what you mean @mpotomin ?

**Comment by TejasQ** [comment]
Yes, @michaeldera. The “Actions” on the table is expected to have a blue outline when it’s focused. It currently doesn’t on `master`. 😊

**Comment by michaeldera** [comment]
Oh... a blue border?

**Comment by TejasQ** [comment]
“Border”, implemented using `box-shadow` as it currently is according to [the spec](https://github.com/contiamo/operational-ui/issues/1006#issuecomment-504932656).

**Comment by michaeldera** [comment]
Just something I noticed while check my PR related to the design of [DataTableInput](https://operational-ui.netlify.com/#/Components/DataTableInput). I thought these look a bit odd because of the corners.

![image](https://user-images.githubusercontent.com/12817839/60050610-4c10a000-96d1-11e9-8ec8-983fbd97077d.png)

![image](https://user-images.githubusercontent.com/12817839/60050659-6fd3e600-96d1-11e9-8983-73bb4feeed22.png)

**Comment by TejasQ** [comment]
@michaeldera that seems odd. Would you be interested to look at solving that case too? Might make more sense to do that in a new PR once we merge your existing one.

**Comment by michaeldera** [comment]
Yes. Should I create a new issue for it?

**Comment by TejasQ** [comment]
Yep. Great idea! It’s almost midnight here in Berlin. I’ll review your pull request first thing in the morning! So excited! 😍

**Comment by michaeldera** [comment]
Sure. Sounds great!

**Comment by kemal-contiamo** [comment]
> Just something I noticed while check my PR related to the design of [DataTableInput](https://operational-ui.netlify.com/#/Components/DataTableInput). I thought these look a bit odd because of the corners.
> 
> ![image](https://user-images.githubusercontent.com/12817839/60050610-4c10a000-96d1-11e9-8ec8-983fbd97077d.png)
> 
> ![image](https://user-images.githubusercontent.com/12817839/60050659-6fd3e600-96d1-11e9-8983-73bb4feeed22.png)

@TejasQ may I remind you that we have a [ticket](https://contiamo.atlassian.net/browse/UI-80) for this issue. Just fyi ;)

**Comment by TejasQ** [comment]
I'm aware, @kemal-contiamo. That internal ticket really doesn't mean anything to OSS contributors though so best to keep it out of issues here. 😄

**Comment by TejasQ** [comment]
The more I think about this, the more I think a 

```
*:focus {
...inputFocus()
}
```

at the global level would be a good idea. I wonder what its drawbacks are.

cc @stereobooster
